site-builder/src/types.rs

```diff
@@ -119,3 +119,10 @@ impl RouteOps {
         matches!(self, RouteOps::Unchanged)
     }
 }
+
+/// Range of bytes for a resource.
+#[derive(Debug, Clone, PartialEq, Eq, Serialize, Deserialize, PartialOrd, Ord)]
+pub struct Range {
+    pub start: Option<u64>,
+    pub end: Option<u64>,
```

Comment:
Since the contract defines start and end as `u256`, why are we using u64 here?

Reply:
changed the contract as well. Since these are byte ranges, `64 bits` is more than enough: `2^64 bytes ~ 18 exabytes` 😅 we will never have blobs this big to index, and this way we save a bit of gas fees.